pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -267,6 +277,26 @@ stdenv.mkDerivation {
       echo "${libc_dev}" > $out/nix-support/orig-libc-dev
     '')
 
+    + ''
+      ##
+      ## General libc++ support
+      ##
+
+    ''
+    + optionalString isClang ''
+      echo "-isystem $out/include/c++/v1" >> $out/nix-support/libcxx-cxxflags
```

Comment:
why is this needed with `isystem ${libcxx}/include/c++/v1` below?

Reply:
also it appears that the /include/c++/v1 thing is only in libc++, not libstdc++

Reply:
(although using libstdc++ with clang is probably unsupported)

Reply:
Isn't the other `$out/include/c++/v1`? I don't know what this is about, to be honest.

Reply:
Oh right, that was me getting confused on the setup hooks. This is indeed wrong.